scripts/coverage_diff.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
```

Comment:
Lines 5 and 6 can be removed, were copy-pasted over by accident from other script.